workshops/smart_calculator/README.md

```diff
@@ -0,0 +1,214 @@
+---
+name: 'Human-Like Calculator Using Python'
+description: 'A calculator where you type a statement and get the answer!'
+author: '@iamsid47'
+img: https://cloud-p6k9fe5b9.vercel.app/0smart-calc.png
+---
+
+## Human-like calculator
+
+In this workshop, we are gonna make a calculator which can add, multiply, divide and subtract two numbers. But the catch is that instead of just adding those two numbers, we give it a statement!
```

Comment:
```suggestion
In this workshop, we are going to make a calculator which can add, multiply, divide and subtract two numbers. But the catch is that instead of just adding those two numbers, we give it a statement!
```

Also what exactly is a "statement"?